main/src/cgeo/geocaching/models/Waypoint.java

```diff
@@ -142,11 +142,12 @@ private int order() {
         return cachedOrder;
     }
 
+    @NonNull
     public String getPrefix() {
         return prefix;
     }
 
-    public void setPrefix(final String prefix) {
+    public void setPrefix(final @NonNull String prefix) {
```

Comment:
Not sure if github sent a notification, but looking at the PR again, the ordering of comments and commits also suggest that you changed the commit after my initial comment.
The seem to have changed how they present that, so probably therefore I got confused somewhat.